src/uebung_2_2.java

```diff
@@ -0,0 +1,18 @@
+
+public class uebung_2_2 {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub

```

Comment:
Der generierte Kommentar sollte in dem Moment gelöscht werden, wo Sie anfangen die Methode mit Code zu füllen.